creator-node/src/config.js

```diff
@@ -658,6 +658,12 @@ const config = convict({
     format: 'BooleanCustom',
     env: 'openRestyCacheCIDEnabled',
     default: false
+  },
+  minimumTranscodingSlotsAvailable: {
```

Comment:
i think this might be for a future pr. either way this value should probably be dynamic and not an env var

Reply:
yup you're right. ill remove this config for this current PR. and sg 👍